addons/sale/models/res_config_settings.py

```diff
@@ -16,7 +16,6 @@ class ResConfigSettings(models.TransientModel):
         config_parameter='sale.use_sale_note')
     group_discount_per_so_line = fields.Boolean("Discounts", implied_group='sale.group_discount_per_so_line')
     module_sale_margin = fields.Boolean("Margins")
-    group_sale_layout = fields.Boolean("Sections on Sales Orders", implied_group='sale.group_sale_layout')
```

Comment:
probably make this setting move from sale to account in a separate commit

Reply:
You're right ! :beer: